jenkins/scripts/integration_test.sh

```diff
@@ -38,7 +38,17 @@ TESTS_FOR="${TESTS_FOR:-integration_test}"
 
 DEFAULT_HOSTS_MEMORY="${DEFAULT_HOSTS_MEMORY:-4096}"
 
+if [ "${DISTRIBUTION}" == "ubuntu" ]
+then
+  BASE_VOLUME_NAME="metal3-ubuntu"
+  TEST_EXECUTER_VM_FLAVOR="2C-4GB-50GB"
```

Comment:
We already have a variable declaration for flavor [here](https://github.com/metal3-io/project-infra/blob/a5671bca76e832e4225f8e6a4f43345a19d91300/jenkins/scripts/integration_test.sh#L50). Is there a need for creating another variable for flavor?

Reply:
And I don't think that 50 GB would be enough for successfully run.

Reply:
> We already have a variable declaration for flavor [here](https://github.com/metal3-io/project-infra/blob/a5671bca76e832e4225f8e6a4f43345a19d91300/jenkins/scripts/integration_test.sh#L50). Is there a need for creating another variable for flavor?



> And I don't think that 50 GB would be enough for successfully run.

Yes, but ideally, should not [this](https://github.com/metal3-io/project-infra/blob/a5671bca76e832e4225f8e6a4f43345a19d91300/jenkins/scripts/integration_test.sh#L50) go to `else` of this [condition](https://github.com/metal3-io/project-infra/blob/a5671bca76e832e4225f8e6a4f43345a19d91300/jenkins/scripts/integration_test.sh#L45)? 
What do you think of memory required to be able to successfully run it?

Reply:
Currently, we are using the same flavor for both Ubuntu & Centos (4C-16GB-100GB). And probably you should still use the same. And once all the installations are moved to a single place, then we can test it again to know if we we can reduce the resource consumption.

Reply:
> > We already have a variable declaration for flavor [here](https://github.com/metal3-io/project-infra/blob/a5671bca76e832e4225f8e6a4f43345a19d91300/jenkins/scripts/integration_test.sh#L50). Is there a need for creating another variable for flavor?
> 
> > And I don't think that 50 GB would be enough for successfully run.
> 
> Yes, but ideally, should not [this](https://github.com/metal3-io/project-infra/blob/a5671bca76e832e4225f8e6a4f43345a19d91300/jenkins/scripts/integration_test.sh#L50) go to `else` of this [condition](https://github.com/metal3-io/project-infra/blob/a5671bca76e832e4225f8e6a4f43345a19d91300/jenkins/scripts/integration_test.sh#L45)?
It can. However, L50 first checks if `TEST_EXECUTER_FLAVOR` var initialized, if yes that is done for departure tests, if not then it will assume that the run is for integration test and assign the default value.

Reply:
Thanks!